lib/nerves_network/dhclient_conf.ex

```diff
@@ -109,24 +111,25 @@ defmodule Nerves.Network.Dhclientv4Conf do
   Start the resolv.conf manager.
   """
   @spec start_link(Path.t, GenServer.options) :: GenServer.on_start
-  def start_link(dhclient_conf_path \\ @dhclient_conf_path, opts \\ []) do
-    GenServer.start_link(__MODULE__, dhclient_conf_path, opts)
+  def start_link([dhclient_conf_path, [name: server_name]], opts \\ []) do
+    IO.puts "#{__MODULE__}: start_link: server_name = #{inspect server_name} dhclient_conf_path = #{inspect dhclient_conf_path} opts = #{inspect opts}"
```

Comment:
Convert this to a log statement or remove

Reply:
done